config/settings.config.php

```diff
@@ -1,203 +1,217 @@
+		// An alternate way of limiting picture upload sizes.
+		// Specify the maximum pixel  length that pictures are allowed to be (for non-square pictures, it will apply to the longest side).
+		// Pictures longer than this length will be resized to be this length (on the longest side, the other side will be scaled appropriately).
+		// If you don't want to set a maximum length, set to -1.
+		'max_image_length' => -1,
+
+		// maximagesize (Integer)
+		// Maximum size in bytes of an uploaded photo.
+		'maximagesize' => 800000,
+
+		// no_regfullname (Boolean)
+		// Allow pseudonyms (true) or enforce a space between first name and last name in Full name, as an anti spam measure (false).
+		'no_regfullname' => true,
+
+		// optimize_max_tablesize (Integer)
+		// Maximum table size (in MB) for the automatic optimization.
+		// -1 to disable automatic optimization.
+		//  0 to use internal default (100MB)
+		'optimize_max_tablesize' => -1,
+
+		// profiler (Boolean)
+		// Enable internal timings to help optimize code. Needed for "rendertime" addon.
+		'profiler' => false,
+
+		// rino_encrypt (Integer)
+		// Server-to-server private message encryption (RINO).
+		// Encryption will only be provided if this setting is set to a non zero value on both servers.
+		// Set to 0 to disable, 2 to enable, 1 is deprecated but won't need mcrypt.
+		'rino_encrypt' => 2,
+
+		// temppath (String)
+		// Custom temporary file directory
+		'temppath' => '',
+
+		// theme (String)
+		// System theme name.
+		'theme' => 'vier',
+
+		// url (String)
+		// The fully-qualified URL of this Friendica node.
+		// Used by the worker in a non-HTTP execution environment.
+		'url' => '',
+	],
+
+	// Used in the admin settings to lock certain features
+	'featurelock' => [
+	],
+
+	// Mapping the log channels to the different log handlers
+	// config entries needs to be set %channelname%.%field%
+	// The only exception is "names"
+	'log_channel' => [
+		'names' => ['index', 'console', 'daemon', 'db', 'auth_ejabbered', 'performance', 'develop'],
+
+		// channels (Array)
+		// The list of handler for this channel
+		'index.handler' => ['default'],
+
+		// errors (Boolean)
+		// Enabling runtime errors, exceptions and fatal errors for this logger
+		'index.errors' => true,
+
+		'console.handler' => ['default'],
+		'console.errors' => true,
+
+		'worker.handler' => ['default'],
+		'worker.errors' => true,
+
+		'daemon.handler' => ['default'],
+		'daemon.errors' => true,
+
+		'db.handler' => ['default'],
+		'db.errors' => true,
+
+		'auth_ejabbered.handler' => ['ejabbered'],
+		'auth_ejabbered.errors' => true,
+
+		'performance.handler' => ['profiler'],
+		'performance.errors' => true,
+
+		'develop.handler' => ['develop'],
+		'develop.errors' => true,
+	],
+
+	// Default log handling in Friendica
+	// config entries needs to be set %handlername%.%field%
+	// The only exception is "names"
+	'log_handler' => [
+		'names' => ['default', 'ejabbered', 'profiler', 'develop'],
```

Comment:
This is also an array, so we might just jump in the deep end and use arrays everywhere. Or convert this to a comma-separated string list.